files/default/test-neutron-ha-tool.py

```diff
@@ -102,22 +102,23 @@ class TestL3AgentMigrate(unittest.TestCase):
     def test_no_dead_agents_returns_zero(self):
         neutron_client = make_neturon_client(live_agents=2)
 
-        result = ha_tool.l3_agent_migrate(neutron_client)
+        result = ha_tool.l3_agent_migrate(neutron_client, None)
```

Comment:
given we have no dead agents, the picker will never be called as no migration will happen.